src/chainstate/burn/operations/test_helpers.rs

```diff
@@ -0,0 +1,89 @@
+use rand::rngs::StdRng;
```

Comment:
Please put this into the file `src/chainstate/burn/operations/test/mod.rs`.

Reply:
Alternatively, you can put this as a `mod test { ... }` in `src/chainstate/burn/operations/mod.rs`.

Reply:
Done :heavy_check_mark: